ansible/roles/schulcloud-calendar/templates/configmap_file_init_db.yml.j2

```
@@ -11,4 +11,8 @@ data:
     git clone https://github.com/hpi-schul-cloud/schulcloud-calendar.git
     cd schulcloud-calendar
     export PGPASSWORD=$DB_PASSWORD
+    until psql -h $DB_HOST -U $DB_USERNAME -d $DB_HOST -c "select 1" > /dev/null 2>&1; do
+      echo "Waiting for postgres server..."
+      sleep 1
+    done
```

Do not work how it shoud be.
The PostgreSQL Server requiers an password.
```suggestion
    echo $DB_PASSWORD > .pgpass
    until psql -h $DB_HOST -U $DB_USERNAME -w -d $DB_HOST -c "select 1" > /dev/null 2>&1; do
      echo "Waiting for postgres server..."
      sleep 1
    done
```

---

this works because the passsword is already set in the env vars, otherwise line 18 would fail